crates/primitives/starknet/src/block/mod.rs

```diff
@@ -30,7 +30,7 @@ pub enum BlockTransactions {
     /// Only hashes
     Hashes(BoundedVec<Felt252Wrapper, MaxTransactions>),
     /// Full transactions
-    Full(BoundedVec<Transaction, MaxTransactions>),
+    Full(BoundedVec<(Transaction, TransactionReceiptWrapper), MaxTransactions>),
```

Comment:
I'm really not sure about this.
https://alpha-mainnet.starknet.io/feeder_gateway/get_block?blockNumber=latest

The receipts seem to be in a separate field of the `Block` struct. Not stored together with the transaction.

@LucasLvy any insight?

Reply:
Yes they are different and are not in the same field

Reply:
Last commit separates the two entities in the block. :+1: